.gitignore

```diff
@@ -12,3 +12,5 @@ python/**/*.sw[ponm]
 python/build/
 python/dist/
 python/sdist/
+
+.idea/
```

Comment:
imo this should be in your local `~/.gitignore`

Reply:
Done.